packages/instant-apps-components/src/assets/t9n/instant-apps-social-share/resources.json

```diff
@@ -27,12 +27,12 @@
       "tooltip": "Click to share to your Facebook account"
     },
     "twitter": {
-      "label": "Twitter",
-      "tooltip": "Click to share to your Twitter account"
+      "label": "X",
```

Comment:
the label is hardcoded in the source code since brand names shouldn't be translated 

https://github.com/Esri/instant-apps-components/blob/master/packages/instant-apps-components/src/components/instant-apps-social-share/instant-apps-social-share.tsx#L429

Reply:
Had a local branch for this one. Here's the PR for it: 
https://github.com/Esri/instant-apps-components/pull/222